sql/moz-fx-data-shared-prod/static/country_codes_v1/metadata.yaml

```diff
@@ -4,3 +4,7 @@ description: |
 
   Includes XK/XKK for Kosovo, which is not in the ISO 3166 standard, but is recognized
   by some governing bodies and is present in MaxMind's GeoIP2 database.
+workgroup_access:
```

Comment:
Something to watch out for is https://github.com/mozilla/bigquery-etl/issues/5830 but since `static` is a special case dataset and we're unlikely to deprecate anything there this propagate without issue.